web/templates/left_sidebar.hbs

```diff
@@ -1,69 +1,76 @@
 <div class="left-sidebar" id="left-sidebar" role="navigation">
-    <div class="narrows_panel">
-        <ul id="global_filters" class="filters">
+    <div id="left_sidebar_navigation_area">
+        <div id="views_label_container">
```

Comment:
Just to reiterate: these new classes and IDs should be separated with hyphens, not underscores.